protos/joystick/joystick.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package mavsdk.rpc.joystick;
+
+option java_package = "io.mavsdk.joystick";
+option java_outer_classname = "JoystickProto";
+
+service JoystickService
+{
+
+    rpc ConnectDrone (ConnectRequest)  returns (ConnectResponse) {}
+    /*
+     * Send command to enable Joystick mode 
+     *
+     * Enabling Joystic mode causes to change into Position Control mode
+     * drone still needs to be armed
+     * Needs to check if the drone is not currently flying or in mission if so drone needs to be paused first 
+     */
+    
+    rpc EnableJoystickMode (EnableRequest)  returns(EnableResponse) {}
+
+
+
+    rpc ArmDrone (ArmRequest) returns (ArmResponse) {}
+
+
+    rpc DisarmDrone (DisarmRequest) returns (DisarmResponse) {}
```

Comment:
We can re-use `Arm` and `Disarm` from the action plugin.

Reply:
Do I delete this or keep it ?

Reply:
I would vote for deleting. No need to duplicate the features, and those ones exist in the `action` plugin.

Reply:
Ok I will delete it